loader/loader.go

```diff
@@ -1005,6 +1005,10 @@ var transformSize TransformerFunc = func(value interface{}) (interface{}, error)
 	switch value := value.(type) {
 	case int:
 		return int64(value), nil
+	case int64:
+		return value, nil
+	case types.UnitBytes:
+		return value, nil
```

Comment:
nit: It think this may work
```suggestion
	case int64, types.UnitBytes:
		return value, nil
```
?